examples/newrelic-infrastructure/README.md

```diff
@@ -28,7 +28,7 @@ translates to
 NRIA_LOG_FILE=/var/log/nr-infra.log
 ```
 
-Edit examples/newrelic-infrastructure/nrconfig.env and configure relevant environment variables for your NewRelic Infrastructure agent.  There are a few defauts defined, but the only required variable is the New Relic license key.
+Edit examples/newrelic-infrastructure/nrconfig.env and configure relevant environment variables for your NewRelic Infrastructure agent.  There are a few defaults defined, but the only required variable is the New Relic license key.
```

Comment:
This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-b571327e4d81ebd5f8f3900f3ee0c00a. Closing.

/close